doc/GuideForCommitters.md

```diff
@@ -0,0 +1,106 @@
+<!--
+Copyright (c) 2019, 2019 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+
+# Eclipse OpenJ9 Committer Guide
+
+This document outlines some general rules for Eclipse OpenJ9 committers to
+follow when reviewing and merging pull requests and issues.  It also provides
+a checklist of items that committers must ensure have been completed prior to
+merging a pull request.
+
+
+## General Guidelines
+
+* Committers should not merge their own pull requests.
+
+* If a pull request modifies the [Contribution Guidelines](https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md),
+request the author to post a detailed summary of the changes on the
+`openj9-dev@eclipse.org` mailing list after the pull request is merged.
+
+* Do not merge a pull request if the Assignee is someone other than yourself.
```

Comment:
An issue or pull request can have multiple assignees, suggest:
'Do not merge a pull request if you are not one of the Assignees'